.changeset/hungry-mails-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Introduce support for local D1 bindings. Enhance development experience by allowing direct integration with `astro dev`.
```

Comment:
```suggestion
Introduces support for local D1 bindings. Enhances development experience by allowing direct integration with `astro dev`.
```
Only an example, totally not a required change, but my *personal* preferred verb tense for these statements is the present tense (not imperative).

And if not full sentences on their own, choose the tense as if it completed the phrase, "This update... " or "The changes in this update...."

e.g. "This update introduces support for local D1 bindings..." If you wanted a standard way of thinking of it!